### ISSUE TITLE
Add file checksum to `/rpc/get-file-url` response

### ISSUE THREAD

**Issue by lkuchenb** [ghga-de/datameta#454]
It would be great to have the file checksum included in the `/rpc/get-file-url` response, which currently looks as follows:

```json
{
  "fileId": {
    "uuid": "string",
    "site": "string"
  },
  "fileUrl": "string",
  "expires": "2021-07-01T07:38:47.227Z"
}
```

**Comment by lkuchenb** [comment]
@nkleinbo we're tracking the requested feature here.

**Comment by lkuchenb** [comment]
Implemented in #455
